app/src/main/java/org/openobservatory/ooniprobe/fragment/measurement/TelegramFragment.java

```diff
@@ -41,7 +41,7 @@ public static TelegramFragment newInstance(Measurement measurement) {
 		application.setText(measurement.getTestKeys().getTelegramEndpointStatus());
 		if (Boolean.TRUE.equals(measurement.getTestKeys().telegram_http_blocking) || Boolean.TRUE.equals(measurement.getTestKeys().telegram_tcp_blocking))
 			application.setTextColor(ContextCompat.getColor(getActivity(), R.color.color_yellow9));
-		webApp.setText(measurement.getTestKeys().getTelegramWebStatus(getActivity()));
+		webApp.setText(measurement.getTestKeys().getTelegramWebStatus());
```

Comment:
What was the reason for this change? I could not find reference to it in the git history.

Reply:
just that param is not used